package-lock.json

```diff
@@ -8119,7 +8119,7 @@
     },
     "diffable-html": {
       "version": "4.0.0",
-      "resolved": "https://registry.npmjs.org/diffable-html/-/diffable-html-4.0.0.tgz",
+      "resolved": "https://npm.morph.int.tools.bbc.co.uk/diffable-html/-/diffable-html-4.0.0.tgz",
```

Comment:
Looks good, just wondering if this change is needed along with the one below for `jest-serializer-html` ?

Reply:
For some reason, those were updated after running `npm ci`.